pkg/autoops/api/progressive_rollout.go

```diff
@@ -65,6 +76,13 @@ func (s *AutoOpsService) CreateProgressiveRollout(
 		return nil, dt.Err()
 	}
 	err = s.mysqlClient.RunInTransaction(ctx, tx, func() error {
+		// There are two reasons of validating auto ops rules:
+		// 1. We have to remove some test cases in `TestCreateProgressiveRollout`.
```

Comment:
I didn't understand the connection between removing the `TestCreateProgressiveRollout` and validating it inside the transaction.